pyas2lib/constants.py

```diff
@@ -28,3 +28,7 @@
     "aes_192_cbc",
     "aes_256_cbc",
 )
+SIGNATUR_ALGORITHMS = (
+    "rsassa_pkcs1v15",
```

Comment:
did the original issue not want support for non rsa algos?

Reply:
Indeed, but the ticket has actually two parts, I only tackled the part that signature algo cannot be passed down, because it is missed in the partner object. 

I miss the information about which other algos we should implement. I do think we can keep ticket as such open and ask that question and make another feature.

Reply:
@abhishek-ram : good to go or ?